package.json

```diff
@@ -19,7 +19,7 @@
   "license": "MIT",
   "dependencies": {
     "rimraf": "^2.6.2",
-    "scuttlebot": "^10.5.0",
+    "scuttlebot": "^11.0.0",
```

Comment:
waiting on https://github.com/ssbc/scuttlebot/pull/473 to resolve and then will use that version number